pisek/utils/text.py

```diff
@@ -15,28 +15,29 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import sys
+from typing import NoReturn
 
 from pisek.utils.colors import ColorSettings
 
 
-def tab(text: str, tab_str: str = "  "):
+def tab(text: str, tab_str: str = "  ") -> str:
     return tab_str + text.replace("\n", f"\n{tab_str}")
 
 
-def pad(text: str, length: int, pad_char: str = " "):
+def pad(text: str, length: int, pad_char: str = " ") -> str:
     return text + (length - len(text)) * pad_char
 
 
-def pad_left(text: str, length: int, pad_char: str = " "):
+def pad_left(text: str, length: int, pad_char: str = " ") -> str:
     return pad(text[::-1], length, pad_char)[::-1]
 
 
-def eprint(msg, *args, **kwargs):
+def eprint(msg, *args, **kwargs) -> NoReturn:
```

Comment:
```suggestion
def eprint(msg, *args, **kwargs) -> None:
```
`NoReturn` means exit from program. In this case `eprint` returns None.